src/cmds/qsub.c

```diff
@@ -236,6 +236,7 @@ static char *pbs_hostvar = NULL; /* buffer containing ",PBS_O_HOST=" and host na
 static int pbs_o_hostsize = sizeof(",PBS_O_HOST=") + 1; /* size of prefix for hostvar */
 static char *display; /* environment variable DISPLAY */
 
+static int is_background = 0; /* flag to check if background qsub is active */
```

Comment:
Change the comment to something like "am i the background process? This variable is set only once and is read-only afterwards"